stylesheet.css

```diff
@@ -0,0 +1,175 @@
+body {
+  background-color: #e5e5e5;
+  background-image: url("image/Topcircles.png");
+  font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif, "Roboto";
+  margin: auto;
+}
+
+.outer-maincont {
+  margin: auto;
+  background-color: #191765;
+  width: 375px;
+  min-height: 100vh;
+  padding-bottom: 10px;
+  color: white;
+}
```

Comment:
AS requested by the previous reviewer please make sure your site is responsive and does not have a fixed with like `width: 375px;` rather use display flex, which will make the page responsive
![image](https://user-images.githubusercontent.com/97521557/198692609-a5e06768-ed6e-4f47-8323-1c20e4759b60.png)